pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/QueryQuotaClusterIntegrationTest.java

```diff
@@ -169,14 +193,60 @@ private void runQueries(double qps, boolean shouldFail)
     assertTrue((failCount == 0 && !shouldFail) || (failCount != 0 && shouldFail));
   }
 
+  private void verifyQuotaUpdate(long quotaQps) {
+    String query = "SELECT COUNT(*) FROM " + getTableName();
+    if (quotaQps == 0) {
+      TestUtils.waitForCondition(aVoid -> !executeQueryOnBroker(query).getExceptions().elements().hasNext()
```

Comment:
looks like the two stmts are same. is this intended to run the query twice back to back?

Reply:
perhaps we can add a broker restful API to get runtime quotas for a table, which can help simplify the test here and can be helpful for debugging in prod as well.

Reply:
> looks like the two stmts are same. is this intended to run the query twice back to back?

Yes, basically making sure consecutive queries pass which indicated quota is not applied.

> perhaps we can add a broker restful API to get runtime quotas for a table, which can help simplify the test here and can be helpful for debugging in prod as well.

That's a good idea, will use this approach.

Reply:
PR to introduce the debug endpoints on broker https://github.com/apache/pinot/pull/13864